contracts/ColonyTask.sol

```diff
@@ -298,10 +304,10 @@ contract ColonyTask is ColonyStorage {
     return taskWorkRatings[_id].secret[_role];
   }
 
-  function setTaskManagerRole(uint256 _id, address payable _user) public
+  function setTaskManagerRole(uint256 _id, address payable _user, uint256 _permissionDomainId, uint256 _childSkillIndex) public
```

Comment:
Just on ordering of parameters. In most places you start off with the domain permission input params but here they are at the end. I don't mind which way it's done, as long as it's consistent.